packages/server/lib/gui/events.ts

```diff
@@ -23,141 +23,13 @@ const handleEvent = function (options, bus, event, id, type, arg) {
 
   switch (type) {
     case 'launch:browser':
-      // TIM: Commented out for reference w/ the new implementation
```

Comment:
@tgriesser do we still need the code commented out here?

Reply:
No, probbaly not